libs/api/ebpf_api.cpp

```diff
@@ -2051,3 +2051,19 @@ ebpf_get_program_type_by_name(
 
     return EBPF_SUCCESS;
 }
+
+_Ret_maybenull_z_ const char*
+ebpf_get_program_type_name(_In_ const ebpf_program_type_t* program_type)
```

Comment:
How will this API work for new program types which are added later? Will we be querying from registry?

Reply:
Correct, issue #223